public/scripts/dashboard.js

```diff
@@ -1,63 +1,80 @@
+            document.getElementById("totalDevelopmentTask").innerHTML =
+                element.totalDevelopmentTask;
+            document.getElementById("totalBusinessTask").innerHTML =
+                element.totalBusinessTask;
+            document.getElementById("totalMarketingTask").innerHTML =
+                element.totalMarketingTask;
+            document.getElementById("totalCompletedTask").innerHTML =
+                element.totalDevelopmentTask +
+                element.totalBusinessTask +
+                element.totalMarketingTask;
             document.getElementById("sprintId").innerHTML = element.sprintId;
             document.getElementById("sprintDetail").innerHTML = element.sprintDetail;
-            document.getElementById("sprintStartDate").innerHTML = element.sprintStartDate;
-            document.getElementById("sprintEndDate").innerHTML = element.sprintEndDate;
-            document.getElementById("sprintEndDate").innerHTML = element.sprintEndDate;
+            document.getElementById("sprintStartDate").innerHTML =
+                element.sprintStartDate;
+            document.getElementById("sprintEndDate").innerHTML =
+                element.sprintEndDate;
+            document.getElementById("sprintEndDate").innerHTML =
+                element.sprintEndDate;
             document.getElementById("sprintStatus").innerHTML = element.sprintStatus;
 
+            getTasks();
         }
     });
-
 }
 
 function sprintFilter(sprintFilterId) {
-    dashboardDataset.forEach(element => {
+    dashboardDataset.forEach((element) => {
         if (parseInt(element.sprintId) == parseInt(sprintFilterId)) {
-            document.getElementById("totalDevelopmentTask").innerHTML = element.totalDevelopmentTask;
-            document.getElementById("totalBusinessTask").innerHTML = element.totalBusinessTask;
-            document.getElementById("totalMarketingTask").innerHTML = element.totalMarketingTask;
-            document.getElementById("totalCompletedTask").innerHTML = element.totalMarketingTask;
-            document.getElementById("totalCompletedTask").innerHTML = element.totalDevelopmentTask + element.totalBusinessTask + element.totalMarketingTask;
+            document.getElementById("totalDevelopmentTask").innerHTML =
+                element.totalDevelopmentTask;
+            document.getElementById("totalBusinessTask").innerHTML =
+                element.totalBusinessTask;
+            document.getElementById("totalMarketingTask").innerHTML =
+                element.totalMarketingTask;
+            document.getElementById("totalCompletedTask").innerHTML =
+                element.totalMarketingTask;
+            document.getElementById("totalCompletedTask").innerHTML =
+                element.totalDevelopmentTask +
+                element.totalBusinessTask +
+                element.totalMarketingTask;
```

Comment:
Indentation issue please talk to shivam for js formatter extension, and fix them